packages/astro/src/components/FooterNav.astro

```diff
@@ -31,86 +32,60 @@ const routes = [
       ["https://www.facebook.com/codeforsanfrancisco", "facebook", true],
       ["https://www.linkedin.com/company/18115347/", "linkedin", true],
       ["https://github.com/sfbrigade/", "github", true],
-      ["http://c4sf.me/slack", "slack", true],
+      ["https://c4sf.me/slack", "slack", true],
       ["https://www.meetup.com/sfcivictech/", "meetup", true],
     ],
   ],
 ] as const;
-// because of annoying differences between dev and build modes, due to this
-// behavior (https://github.com/withastro/astro/issues/5630), always remove
-// any trailing slash so currentPage can match the pages above
-const fullPath = Astro.url.pathname.replace(/\/$/, "");
-const currentPage = fullPath.slice(fullPath.lastIndexOf("/") + 1);
 ---
 
-<nav class="container">
-  <ul class="grid footer-list container">
-    {
-      routes.map(([cat, links]) => (
-        <li class="footer-list-item">
-          <h3 class="footer-list-cat-name contrast">{cat}</h3>
-          <ul class="footer-list-cat-list">
-            {links.map(([page, label, needsIcon]) => (
-              <li class="footer-list-cat-list-item container">
-                {needsIcon ? (
-                  <span class="icon-container">
-                    <Icon name={"fa:" + label} />{" "}
-                  </span>
-                ) : (
-                  ""
-                )}{" "}
-                <a
-                  href={page}
-                  aria-current={currentPage === page ? "page" : false}
-                >
-                  {label}
-                </a>
-              </li>
-            ))}
-          </ul>
-        </li>
-      ))
-    }
-  </ul>
-</nav>
+<footer class="container">
+	<nav>
+		{columns.map(([cat, links]) => (
```

Comment:
Can we rewrite the data structure on this page as a collection of objects instead of the nested array format? I think its a little bit more readable and safer to maintain. We would probably need to change this bit of code too in that case.

Reply:
Sure.  I think Nick was just following my original setup in the header.  But the header/footer links will likely end up being specified in the CMS, not in code, so they're easy to change.